app/services/tickets/v1/create.rb

```diff
@@ -0,0 +1,21 @@
+module Tickets::V1
+  class Create
+		attr_accessor :title, :description, :ticket
+
+    def initialize(params)
+      @current_user = params[:current_user]
+      @title = params[:title]
+      @description = params[:description]
+    end
+
+		def call
+      save
```

Comment:
change method name to - create_ticket

Reply:
@varungadde99 actually name of the service is create so function name is save